lib/active_model/serializer/reflection.rb

```diff
@@ -42,17 +42,17 @@ def initialize(*)
 
       def link(name, value = nil, &block)
         @_links[name] = block || value
-        nil
+        :nil
```

Comment:
Maybe create an issue or start a pr to discuss any improvements 

B mobile phone

> On Feb 12, 2016, at 6:11 PM, Yohan Robert <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/reflection.rb:
> 
> > @@ -42,17 +42,17 @@ def initialize(*)
> >  
> >        def link(name, value = nil, &block)
> >          @_links[name] = block || value
> > -        nil
> > +        :nil
> Yes. For more details, there are two use cases to consider for the relationship block:
> 
> Where the block return the value we want to use for the relationship:
> has_one :author do
>   meta some: :thing
>   object.own_author
> end
> Where the block return some value to define that we want to use the default relationship:
> has_one :author do
>   meta some: :thing
> end
> For the latest, I defined the value to be :nil but it could be change to anything else if a better suggestion pops up since this is the internal API.
> 
> —
> Reply to this email directly or view it on GitHub.
>